barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -1286,7 +1286,7 @@ void prove_honk_output_all(const std::string& bytecodePath,
     using VerificationKey = Flavor::VerificationKey;
 
     bool honk_recursion = false;
-    if constexpr (IsAnyOf<Flavor, UltraFlavor>) {
+    if constexpr (IsAnyOf<Flavor, UltraFlavor, UltraKeccakFlavor>) {
```

Comment:
Just pointing this change out! Without it the proofs generated by `prove_ultra_keccak_honk_output_all` couldn't be verified.